waffle/south_migrations/0001_initial.py

```diff
@@ -46,9 +50,9 @@ def forwards(self, orm):
         db.create_table('waffle_flag_users', (
             ('id', models.AutoField(verbose_name='ID', primary_key=True, auto_created=True)),
             ('flag', models.ForeignKey(orm['waffle.flag'], null=False)),
-            ('user', models.ForeignKey(get_user_model(), null=False))
+            (User._meta.model_name, models.ForeignKey(orm[user_orm_label], null=False))
```

Comment:
@combitel Why did you make the name of the `user` field dynamic? Isn't this an arbitrary, waffle-internal name that may as well stay hardcoded? If yes, then the 1.4 test failure will be resolved and this PR ready to go. Which would be great, since we have the issue discussed in #152 and fixed here, too. Thanks for the PR! :+1: